### PR TITLE
Feature Concept: recommended bar extra options for ease of discovery

### PR DESCRIPTION
**Pull request by saltrafael**
## PR Checklist

Please check all that apply to this PR using "x":

- [X] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [ ] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number:

## What is the current behavior?

Recommended bar could be improved to ease discovery in less clicks for the user

## What is the new behavior?

Added extra options for the recommended bar to allow for discovering more specific related content based on channel or tags, as shown in the pics:

 

 

Comparison to that one popular Odysee Alternative from Web2.0:

The main difference being that the odysee alternative platform doesn't allow to filter tags

## Other information

This is a concept I've been testing on the weekend, there are some styling issues like if the channel name is too big you can't expand. Also the autoplay will keep playing the recommended content from "All", it won't be affected if you are at the "More from" or tag tabs

### CODE REVIEW

**Comment by tzarebczan** [comment]
Wow, this is a great start and good place to add improvements! For tag searches, are you using claim search?

**Comment by saltrafael** [comment]
> Wow, this is a great start and good place to add improvements! For tag searches, are you using claim search?

Thanks! I just used a claim list discover with the selected tag as filter, so it's the same as going to each tags page, but faster and with less clicks

**Comment by tzarebczan** [comment]
Yes, that's awesome! Maybe it can default to trending sort, and then let users change new/top also. 

This would be a great one to add soon also: https://github.com/lbryio/lbry-desktop/issues/5181 (there's some other search improvements in another PR too).

**Comment by saltrafael** [comment]
> Yes, that's awesome! Maybe it can default to trending sort, and then let users change new/top also.
> 
> This would be a great one to add soon also: #5181 (there's some other search improvements in another PR too).

You mean like this?

 

I had hid the filters so it doesn't look too different from before

**Comment by kauffj** [comment]
this is a really cool idea but will need a style/UX pass from @seanyesmunt

File: ui/component/recommendedContent/view.jsx [hunk 1]
```diff
@@ -24,6 +27,14 @@ type Props = {
 
 export default function RecommendedContent(props: Props) {
   const { uri, claim, search, mature, recommendedContent, nextRecommendedUri, isSearching, isAuthenticated } = props;
+  const [expanded, setExpanded] = React.useState(false);
+  const [allRelated, setAllRelated] = React.useState(true);
+  const [moreFrom, setMoreFrom] = React.useState(false);
```
Comment (seanyesmunt) [inline ui/component/recommendedContent/view.jsx:32]: for now lets just start with the "more from" toggle only.

**Review by seanyesmunt — CHANGES_REQUESTED** [review]
This is great! For now lets just start with one category "more from". Then we don't have to worry about expanding the section for more categories. Once those are removed I will push up a commit to do some styling to this new section.

Thank you for the PR! This will be great to have :slightly_smiling_face:

**Comment by saltrafael** [comment]
> This is great! For now lets just start with one category "more from". Then we don't have to worry about expanding the section for more categories. Once those are removed I will push up a commit to do some styling to this new section.
> 
> Thank you for the PR! This will be great to have 🙂

Done, and thanks for the feedback!!

**Comment by seanyesmunt** [comment]
I added your commits to this branch so I could rebase
https://github.com/lbryio/lbry-desktop/pull/5847

Will merge once the build completes

**Pull request closed without merging**